tests/e2e/specs/blocks/content-helper/top-bar-icon.spec.js

```diff
@@ -21,7 +21,7 @@ const pluginButton = 'button[aria-label="Parse.ly Content Helper"]';
  * Tests for the Content Helper's top bar icon.
  */
 describe( 'Content Helper top bar icon in the WordPress Post Editor', () => {
-	const relatedPostsTitle = 'Related top-performing post';
+	const contentHelperTitle = 'Current Post DetailsRelated Top-Performing Posts';
```

Comment:
This doesn't seem quite right?

Reply:
This isn't pretty, but it is the expected value (titles of all Content Helper panels).